coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -130,3 +130,45 @@ def to_spacecase(string):
     return re.sub('(?<=[^\s])([A-Z])',
                   lambda match: ' ' + match.group(1),
                   string)
+
+
+def to_kebabcase(string):
+    """
+    Converts the given string to kebab-case.
+
+    >>> to_kebabcase('HelloWorld')
+    'hello-world'
```

Comment:
Um, I think it does not make sense to have a - preceding the leading Uppercase character if we were to think about this from a point of view of snake case. Let's say if the example was '   String  ', if we were to follow npm convention, it'll be ----string-- rather than ---string-- . What I feel is it's just really unnecessary, and not really serving any purpose.